config/initializers/session_store.rb

```diff
@@ -10,8 +10,10 @@
 servers = ApplicationConfig["REDIS_SESSIONS_URL"] || ApplicationConfig["REDIS_URL"]
 Rails.application.config.session_store :redis_store,
                                        key: ApplicationConfig["SESSION_KEY"],
+                                       domain: ApplicationConfig["APP_DOMAIN"],
                                        servers: servers,
                                        expire_after: expires_after,
                                        signed: true,
                                        secure: ApplicationConfig["FORCE_SSL_IN_RAILS"] == "true",
+                                       same_site: :lax,
```

Comment:
should this be set only if secure is true?

Reply:
What I'm reading is that

> You must ensure that you pair SameSite=None with the Secure attribute

The same doesn't apply for `SameSite=Lax` eh?